addons/crm/models/utm.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api, SUPERUSER_ID
+
+class UtmCampaign(models.Model):
+    # OLD crm.case.resource.type
+    _inherit = 'utm.campaign'
+    _description = 'UTM Campaign'
+
+    crm_lead_activated = fields.Boolean('Use Leads', compute='_compute_crm_lead_activated')
+    global_lead_count = fields.Integer('Lead Count', groups='sales_team.group_sale_salesman', compute="_compute_global_opportunity_and_lead_count")
```

Comment:
NamingThings: is the global necessary ? As you are on a campaign record you know this is global to the campaign.